app/components/viral/time_ago_component.html.erb

```diff
@@ -1,3 +1,3 @@
 <%= viral_tooltip(title: l(original_time, format: :long)) do %>
-  <span class='text-sm text-gray-900 dark:text-white'><%= time_ago %></span>
+  <%= time_ago %>
```

Comment:
**nitpick (blocking):** This is no longer working in lookbook preview, but I see that it still works in the table. Is it now the responsibility of whatever uses it to set the styles?

Reply:
Correct.  If you look at the tables in main, the time\_ago fields do not match the rest of the table.  I updated the lookbook preview in 68b4107